modules/logging-rn-5.5.17.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+// logging-5-5-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-17_{context}"]
+= Logging 5.5.17
+This release includes link:https://access.redhat.com/errata/RHSA-2023:5542[OpenShift Logging Bug Fix Release 5.5.17].
+
+[id="openshift-logging-5-5-17-bug-fixes_{context}"]
+== Bug fixes
+None.
```

Comment:
I would suggest this:

There are no bug fixes in this release.

Reply:
Actually, I'm wondering, if there are none, I don't know if you need to necessarily say None. You can just link to the errata like here: https://docs.openshift.com/container-platform/4.13/security/file_integrity_operator/file-integrity-operator-release-notes.html#file-integrity-operator-release-notes-1-2-1

Reply:
Hi @skrthomas. I'm not including this comment for now to maintain consistency since Logging has been following this style for the Release Notes: https://docs.openshift.com/container-platform/4.12/logging/logging_release_notes/logging-5-5-release-notes.html. I can discuss this with my team if we want to change the style which can be done as a part of restructuring. Thanks! :)

cc @briandooley

Reply:
ACK, no worries I know all these Operators kinda have different release notes styles.